src/widgets/benefits/ui/benefits.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { RenderResult, screen } from '@testing-library/react';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import { Benefits } from '@/widgets/benefits';
+import { benefitMentorshipHome } from 'data';
+
+describe('Benefits', () => {
+  let rendered: RenderResult;
+
+  beforeEach(() => {
+    rendered = renderWithRouter(<Benefits />);
+  });
+
+  it('renders the title', () => {
+    expect(screen.getByText('Mentorship is for you if you')).toBeVisible();
+  });
+
+  it.each(benefitMentorshipHome.benefits || [])('renders the benefit $id with the given info', ({ id, text }) => {
+    const benefit = screen.getAllByTestId('benefit').at(id - 1);
+
+    expect(benefit).toHaveTextContent(text);
+  });
+
+  afterEach(() => {
+    rendered.unmount();
+  });
```

Comment:
I beleave that RTL automatically takes care of unmounting components between tests, so you typically don't need to call the unmount manually

Reply:
Done